src/cityreader/cityreader.py

```diff
@@ -1,11 +1,22 @@
+import csv
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
 
 
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __str__(self):
+        return f"{self.name} {self.lat} {self.lon}"
```

Comment:
Love your use of new and shiny Fstring. fyi, this only works with  v3.6 and later